src/Microsoft.AspNetCore.Authentication/project.json

```diff
@@ -10,6 +10,7 @@
     "keyFile": "../../tools/Key.snk"
   },
   "dependencies": {
+    "Microsoft.AspNetCore.Antiforgery": "1.0.0-*",
```

Comment:
You guys realize this results in a new `services.AddCookieAuthentication()` being required everywhere right?